tests/Nox.Types.Tests/EntityFrameworkTests/Configuration/CountryDbConfiguration.cs

```diff
@@ -46,6 +46,7 @@ public void Configure(EntityTypeBuilder<Country> builder)
         builder.OwnsOne(e => e.LatLong).Ignore(p => p.Value);
         builder.OwnsOne(e => e.GrossDomesticProduct).Ignore(p => p.Value);
         builder.OwnsOne(e => e.DateTimeRange).Ignore(p => p.Value);
+        builder.OwnsOne(e => e.DateTimeSchedule).Ignore(p => p.Value);
```

Comment:
I think we should have a converter, since we are using converter even for simpler types.
We should check with Ricardo which types won't need a Converter.